client/web/src/hooks/useWebsocket.js

```diff
@@ -26,6 +27,13 @@ const useWebsocket = (token, onOpen, onMessage, selectedModel) => {
             var ipAddress = parts[0];
             var currentPort = parts[1];
 
+            // Check if the host is 'localhost', IP address or 'realchar.ai'
+            if (ipAddress === 'localhost' || (isIP(ipAddress) || isIPv4(ipAddress))) {
+                // continue with the current host
+            } else if (ipAddress === 'realchar.ai') {
```

Comment:
Do not hardcode

Reply:
do you mean do not hardcode realchar.ai? or both realchar.ai and localhost